### DISCUSSION TITLE
Student Developer Pack

### DISCUSSION THREAD

**Discussion by 21A35A0509-cse** [community/community#59409] [GitHub Education]
Access to Student benefits

Question

### Body

I applied for Student Developer Pack and my application was approved after several rejections. But i still didn't get access to the pack as i am near to deadlines to complete my project. I request to provide me access to GitHub Student Developer Pack.

**Comment by queenofcorgis** [comment]
Welcome to the GitHub Community, @21A35A0509-cse , we're happy you're here! You are more likely to get a useful response if you are posting your question in the applicable category. I've gone ahead and moved it for you. Good luck!

**Comment by samus-aran** [comment]
Hey @21A35A0509-cse  👋

We've updated our application and re-verification process to ensure that GitHub and our partners can offer exceptional, free benefits to students worldwide during their academic journey.

With this update, when you submit your application, you'll have your academic affiliation verified and gain access to Global Campus right away. Your educational benefits will arrive within 72 hours of this verification. However, during busy periods like the start of a new semester, it may take a bit longer.

You’ll get an email notification once your academic benefits are ready, or you can check your status [here](https://education.github.com/discount_requests/pack_application). Meanwhile, as a verified [GitHub Global Campus](https://education.github.com/globalcampus/student) student, you can explore the platform, connect with a local [Campus Expert](https://education.github.com/students/experts), attend events hosted by student leaders and GitHub Education, discover and contribute to student-created repositories from [GitHub Community Exchange](https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-global-campus-for-students/about-github-community-exchange) and much more!

I'll be closing this discussion due to lack of recent activity. If you still haven't received your Global Campus benefits please feel free to re-open and we'll be happy to assist!
